plugins/Jobs.cpp

```diff
@@ -239,7 +239,7 @@ bool BedrockPlugin_Jobs::peekCommand(SQLite& db, BedrockCommand& command) {
                 }
 
                 // If there's no job or the existing job doesn't match the data we've been passed, escalate to master.
-                if (!result.empty() && result[0][1] == job["data"]) {
+                if (!result.empty() && result[0][1] == (job["data"].empty() ? SToStr("{}") : job["data"])) {
```

Comment:
Ok, I figured it out, I read the whole error in the logs haha It's because of the ternary operator, `"{}"` is `char*` while `job["data"]` is `SString`, it seems both have to be the same type. 

Anyways, should we move the ternary operator to a variable? It's a bit hard to read right now. Having a ternary condition in a comparator seems odd.